db/migrate/20230126135046_add_author_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] You have done a great job adding the user's reference to the comments table. However, this will result in having a user tag/name and according to the project requirements ERD, the post is supposed to reference the user's table as an author. You could instead do it this way: ```add_reference :comments, :author, foreign_key: {to_table: :users}```
![image](https://user-images.githubusercontent.com/86318284/214902439-b8cc05bd-e53f-4e2d-acb1-b286c8e33e8c.png)